sdk/keyvault/keyvault-certificates/src/certificatesModels.ts

```diff
@@ -52,9 +64,63 @@ export interface CertificateWithPolicy extends Certificate {
 
 /**
  * @interface
- * An interface representing the attributes of a certificate
+ * An interface representing a certificate's policy
+ */
+export interface CertificatePolicy
+  extends SecretProperties,
+    X509CertificateProperties,
```

Comment:
>X509CertificateProperties [](start = 4, length = 25)

Do we need a custom interface for this?  X509CertificateProperties doesn't have the refactoring we did on AlternateSubjectNames. Also KeyUsage on X509Properties doesn't support the enum we added for KeyUsages.

Reply:
keyUsage is an array of a limited set of possible string properties (KeyUsageType). It's the typescript enum that works with strings,  right?

Reply:
Can we leave these as x509CertificateProperties inside of "properties"? All of the properties at that level are only valid for the specific x509 usage, which is entirely different with the normal behavior.